js/classes/entities/midground/enemies/village/bunny.js

```diff
@@ -82,13 +67,13 @@ class Bunny {
         // HOW SHOULD WE MOVE HIM BASED ON HIS CONDITIONS?
         const mult = (this.facing === "left") ? -1 : 1;
         if (this.action === "hopping") {
-            this.x += mult * Bunny.SPEED * GAME.clockTick;
+            this.pos = Vector.add(this.pos, new Vector(mult * Bunny.SPEED * GAME.clockTick, 0));
         }
     };
 
     /** Draw the Bunny on the canvas. */
     draw() {
-        this.animations[this.facing][this.action].drawFrame(this.x, this.y, Bunny.SCALE);
+        this.animations[this.facing][this.action].drawFrame(this.pos, Bunny.SCALE);
```

Comment:
The Bunny's position should be converted to canvas space before passing to the `drawFrame` method, e.g.: 
`this.animations[this.facing][this.action].drawFrame(Vector.worldToCanvasSpace(this.pos), Bunny.SCALE);`